src/UI/home/dashboard/chart.tsx

```diff
@@ -31,6 +62,18 @@ export function Chart(props: any) {
             <PieChart
                 className={'chart-container__chart'}
                 data={data}
+                animate={true}
+                animationDuration={500}
+                animationEasing="ease-out"
+                labelPosition={50}
+                lengthAngle={360}
+                lineWidth={100}
+                paddingAngle={0}
+                radius={50}
+                rounded={false}
+                startAngle={0}
+                viewBoxSize={[100, 100]}
+                onFocus={(e, segmentIndex) => data.values()}
```

Comment:
e and segmentIndex are not used, so you don't have to declare them. `() => data.values()` will also work